src/browser/reset.js

```diff
@@ -0,0 +1,60 @@
+import { sendMouse, setViewport } from '@web/test-runner-commands';
+import { nextFrame } from '@open-wc/testing';
+
+const DEFAULT_LANG = 'en',
+	DEFAULT_VIEWPORT_HEIGHT = 800,
+	DEFAULT_VIEWPORT_WIDTH = 800;
+
+let currentLang = undefined,
+	currentRtl = false,
+	currentViewportHeight = 600,
+	currentViewportWidth = 800;
```

Comment:
Does the `currentViewportHeight` purposefully start off different from the default to trigger an initial `setViewport`?

Reply:
Right, should have put a note about this. Even though I can't find anything in the Playwright documentation about it, it looks like the default viewport when you launch it is 800x600. So yes, this is so that when the first test runs it resizes the viewport to 800x800.

Now since the height of the viewport shouldn't really matter at all anymore, maybe we should just change our default height to 600? The only time it would be used would be if we added an API to snapshot the entire page (which we likely will).

Reply:
Hmm, yeah we could do that, although what we have now saves us if Playwright ever decides to change their value.  My thinking was maybe setting both to `0` was clearer

Reply:
I like that -- done.